api/client/stack/common.go

```diff
@@ -48,3 +50,43 @@ func getNetworks(
 		ctx,
 		types.NetworkListOptions{Filters: getStackFilter(namespace)})
 }
+
+type stack struct {
+	// Name is the name of the stack
+	Name string
+}
+
+func getStacks(
+	ctx context.Context,
+	apiclient client.APIClient,
+) ([]*stack, error) {
```

Comment:
I think this would be better in `api/client/stack/list.go`, it's specify to listing stacks.